README.md

```diff
@@ -1 +1,26 @@
-# node-repl-lab
\ No newline at end of file
+# Running JavaScript Code in Node REPL
+
+## Objectives
+
+1. Use REPL
+1. Execute code in REPL
+1. Exit REPL
+
+## Introduction
+
+Understanding global and scoping will give you a good foundation for future development.
```

Comment:
what do you mean by global and scoping? maybe say why it will give you a good foundation - how is it useful?